dimod/constrained/constrained.py

```diff
@@ -1021,32 +1092,40 @@ def from_file(cls,
 
         header_info = read_header(file_like, CQM_MAGIC_PREFIX)
 
-        if not (1, 0) <= header_info.version <= (1, 3):
+        num_variables = header_info.data["num_variables"]
+
+        if not (1, 0) <= header_info.version <= (2, 0):
```

Comment:
Is `1.0` actually supported?

Reply:
Yep, the difference is actually quite small. Agree this need a documentation pass to help make this stuff clearer.